Commands/Fields/RemoveField.cs

```diff
@@ -9,15 +9,19 @@ namespace SharePointPnP.PowerShell.Commands.Fields
     [Cmdlet(VerbsCommon.Remove, "SPOField", SupportsShouldProcess = true)]
     [CmdletHelp("Removes a field from a list or a site",
         Category = CmdletHelpCategory.Fields)]
+    [CmdletExample(
+        Code = @"PS:> Remove-SPOField -List ""Demo list"" -Identity ""Speakers""",
+        Remarks = @"Gets the speakers field from the list Demo list",
+        SortOrder = 1)]
```

Comment:
Maybe good to also provide an example of the cmdlet without the List parameter, which allows one to remove a site column.